mysql-compatibility.md

```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/dev/mysql-compatibility/','/docs/dev/reference/mysql-compatibil
 
 # MySQL Compatibility
 
-TiDB is highly compatible with the MySQL 5.7 protocol and the common features and syntax of MySQL 5.7. The ecosystem tools for MySQL 5.7 (PHPMyAdmin, Navicat, MySQL Workbench, mysqldump, and Mydumper/myloader) and the MySQL client can be used for TiDB.
+TiDB is highly compatible with the MySQL protocol and the common features and syntax of MySQL 5.7 and MySQL 8.0. The ecosystem tools for MySQL (PHPMyAdmin, Navicat, MySQL Workbench, mysqldump, and Mydumper/myloader) and the MySQL client can be used for TiDB.
```

Comment:
Note that on https://docs.pingcap.com/tidb/stable/dev-guide-third-party-support#gui we have this:

![image](https://github.com/pingcap/docs/assets/1272980/e955ed54-a7bf-4d48-b1bf-dd38cd6d2518)

```suggestion
TiDB is highly compatible with the MySQL protocol and the common features and syntax of MySQL 5.7 and MySQL 8.0. The ecosystem tools for MySQL (PHPMyAdmin, Navicat, MySQL Workbench, DBeaver and more) and the MySQL client can be used for TiDB.
```

There are now even more VS Code plugins that work with TiDB: https://github.com/mtxr/vscode-sqltools/pull/1113 and https://github.com/dbcli/mycli also has good TiDB support now.

I don't think Mydumper/myloader are very important now as we have Dumpling/Lightning.

MySQL Shell and MySQL Client both also work for running SQL queries etc.

Reply:
@dveeden  Good point! In addition, I'd like to include the link of 3rd-party tool here.  We shall file another PR to refine the [Third-Party Tools](https://docs.pingcap.com/tidb/stable/dev-guide-third-party-support), and append mycli. 

```suggestion
TiDB is highly compatible with the MySQL protocol and the common features and syntax of MySQL 5.7 and MySQL 8.0. The ecosystem tools for MySQL (PHPMyAdmin, Navicat, MySQL Workbench, DBeaver and [more](/dev-guide-third-party-support#gui) ) and the MySQL client can be used for TiDB.
```